src/Chorus.Tests/Chorus.Tests.csproj

```diff
@@ -28,7 +28,7 @@
 
   <ItemGroup>
     <PackageReference Include="GitVersionTask" Version="5.3.4" PrivateAssets="All" />
-    <PackageReference Include="L10NSharp" Version="4.1.0-beta0047" />
+    <PackageReference Include="L10NSharp" Version="*-*" />
```

Comment:
I think it would be better to specify a minimum version so that we have some documentation of a version that successfully built, i.e. `4.1.0*-*`.

(For reference mainly for myself: https://docs.microsoft.com/en-us/nuget/concepts/package-versioning)

Reply:
good idea; done